src/mail-app/search/SearchBar.ts

```diff
@@ -123,7 +123,8 @@ export class SearchBar implements Component<SearchBarAttrs> {
 			"form.full-width",
 			{
 				style: {
-					"max-width": styles.isUsingBottomNavigation() ? "" : px(350),
+					"min-width": styles.isUsingBottomNavigation() ? "" : px(size.second_col_max_width),
```

Comment:
if we set both min width and max width and they are set to the same value would it make more sense to just set width? or is there a situation where we need to change it?
also this one has `.full-width` which is getting overwritten I guess